docs/reference/aggregations/metrics/percentile-aggregation.asciidoc

```diff
@@ -249,7 +249,12 @@ it. It would not be the case on more skewed distributions.
 [[search-aggregations-metrics-percentile-aggregation-compression]]
 ==== Compression
 
+ifdef::asciidoctor[]
+experimental["The `compression` parameter is specific to the current internal implementation of percentiles, and may change in the future"]
```

Comment:
Could you do `experimental::[` here? That is the "normal" way of doing these in asciidoctor. The way without the `[` is a compatibility thing we've added that we'd like to phase out. Not for older versions, but eventually. So I think it is nice to get in the habit of doing it.